src/dynamics/mpas/dp_coupling.F90

```diff
@@ -203,10 +225,11 @@ subroutine p_d_coupling(phys_state, phys_tend, dyn_in)
 
    ! CAM physics output redistributed to blocks.
    real(r8), allocatable :: t_tend(:,:)
-   real(r8), allocatable :: qv_tend(:,:)
+   real(r8), allocatable :: q_tend(:,:,:)
```

Comment:
The variable only contains thermodynamically active species and not all advected species (in a standard F2000 AMIP setup the thermodynamic active tracers  are water vapor and condensates; for WACCM-x it would include components of dry air). I am fine with renaming if you wish.

Reply:
I think tracer would be more standard.  That name is used throughout CAM including the other dycores (and I see it used a lot in MPAS that way as well). It used to be called the q array in CAM but it is being standardized as tracer. If that's OK with everyone I will rename to tracer_tend

Reply:
tracer_tend sounds good to me

Reply:
It is only the thermodynamically active species but that would make the name really long: thermodynamically_active_tracer_tend

I am fine with what you choose.